### PR TITLE
Wallaby bump snapshot 01092022

### PR DESCRIPTION
**Pull request by grzegorzkoper**
Hosts and containers updated on sms-lab. Triggering promotion to "production"

### CODE REVIEW

File: etc/kayobe/kolla/globals.yml [hunk 1]
```diff
@@ -4,29 +4,29 @@ docker_yum_baseurl: "{{ stackhpc_repo_docker_url }}"
...
-ovn_tag: wallaby-20220223T143249
-openvswitch_tag: wallaby-20220223T143249
-prometheus_tag: wallaby-20220525T174700
-prometheus_jiralert_tag: wallaby-20220119T122428
-prometheus_libvirt_exporter_tag: wallaby-20220325T122042
-prometheus_msteams_tag: wallaby-20220119T122428
-prometheus_openstack_exporter_tag: wallaby-20220705T132206
-skydive_agent_tag: wallaby-20220817T151053
-skydive_analyzer_tag: wallaby-20220817T151053
+#barbican_tag: wallaby-20220311T133847
```
Comment (markgoddard) [inline etc/kayobe/kolla/globals.yml]: Could you remove them?

Comment (grzegorzkoper) [reply etc/kayobe/kolla/globals.yml]: done.

File: etc/kayobe/kolla/globals.yml [hunk 2]
```diff
@@ -4,29 +4,29 @@ docker_yum_baseurl: "{{ stackhpc_repo_docker_url }}"
...
+#bifrost_tag: wallaby-20220825T103439
```
Comment (markgoddard) [inline etc/kayobe/kolla/globals.yml]: Actually, we'll still need bifrost here. Did you build a new bifrost image?

File: etc/kayobe/kolla/globals.yml [hunk 3]
```diff
@@ -2,31 +2,3 @@
...
-openvswitch_tag: wallaby-20220223T143249
-prometheus_tag: wallaby-20220525T174700
-prometheus_jiralert_tag: wallaby-20220119T122428
-prometheus_libvirt_exporter_tag: wallaby-20220325T122042
-prometheus_msteams_tag: wallaby-20220119T122428
-prometheus_openstack_exporter_tag: wallaby-20220705T132206
-skydive_agent_tag: wallaby-20220817T151053
-skydive_analyzer_tag: wallaby-20220817T151053
-{% else %}
-bifrost_tag: wallaby-20220825T112231
```
Comment (markgoddard) [inline etc/kayobe/kolla/globals.yml:31]: Need this `else` for Ubuntu

File: etc/kayobe/kolla/globals.yml [hunk 4]
```diff
@@ -2,31 +2,3 @@
 ---
 docker_yum_baseurl: "{{ stackhpc_repo_docker_url }}"
 docker_yum_gpgkey: "https://download.docker.com/linux/{% raw %}{{ ansible_facts.distribution | lower }}{% endraw %}/gpg"
-
-{% if kolla_base_distro == 'centos' %}
-barbican_tag: wallaby-20220311T133847
-bifrost_tag: wallaby-20220825T103439
```
Comment (markgoddard) [inline etc/kayobe/kolla/globals.yml:8]: and need Bifrost here for CentOS. Did you build a bifrost (seed) image?

Comment (grzegorzkoper) [reply etc/kayobe/kolla/globals.yml:8]: didn't build seed images, will run new job to do it

File: etc/kayobe/kolla/globals.yml [hunk 5]
```diff
@@ -3,30 +3,7 @@
 docker_yum_baseurl: "{{ stackhpc_repo_docker_url }}"
 docker_yum_gpgkey: "https://download.docker.com/linux/{% raw %}{{ ansible_facts.distribution | lower }}{% endraw %}/gpg"
 
-{% if kolla_base_distro == 'centos' %}
-barbican_tag: wallaby-20220311T133847
+% if kolla_base_distro == 'centos' %}
```
Comment (markgoddard) [inline etc/kayobe/kolla/globals.yml]: Missing {

File: etc/kayobe/kolla/globals.yml [hunk 6]
```diff
@@ -3,30 +3,7 @@
...
-prometheus_tag: wallaby-20220525T174700
-prometheus_jiralert_tag: wallaby-20220119T122428
-prometheus_libvirt_exporter_tag: wallaby-20220325T122042
-prometheus_msteams_tag: wallaby-20220119T122428
-prometheus_openstack_exporter_tag: wallaby-20220705T132206
-skydive_agent_tag: wallaby-20220817T151053
-skydive_analyzer_tag: wallaby-20220817T151053
 {% else %}
 bifrost_tag: wallaby-20220825T112231
-{% endif %}
```
Comment (markgoddard) [inline etc/kayobe/kolla/globals.yml:32]: Missing endif

File: etc/kayobe/kolla/globals.yml (hunk 5 above)
Comment (markgoddard) [reply etc/kayobe/kolla/globals.yml]: Still

Comment (grzegorzkoper) [reply etc/kayobe/kolla/globals.yml]: fixed

File: etc/kayobe/kolla/globals.yml (hunk 6 above)
Comment (grzegorzkoper) [reply etc/kayobe/kolla/globals.yml:32]: fixed

**Review by markgoddard — APPROVED** [review]

**Pull request merged**